src/wp-includes/class-wp-scripts.php

```diff
@@ -871,8 +883,22 @@ private function get_dependents( $handle ) {
 	 */
 	private function get_intended_strategy( $handle ) {
```

Comment:
@kt-12 I'm not really sure on the best practices around this and whether my comment is valid, but we do not check for the presence of `$handle` before using it in the method body. Given that this is a private method and not really intended to be called ad hoc by the _public_, it's probably not an issue, correct?

Reply:
I suppose this same comment applies to my previous comment around defining a default param value.

Reply:
@10upsimon, when it comes to `$handle`, it will come from `wp_register_script` or `wp_enqeue_script` or from core scripts, all of which have `$handle` as non-optional param.

Also, this function is only called via `get_elegible_loading_strategy`, which has a check for registered handles and returns `''` if not registered. Since this is a private function, we can be certain that it won't be called in a way that it breaks.